lib/rspec/enqueue_sidekiq_job.rb

```diff
@@ -126,9 +129,26 @@ def supports_value_expectations?
 
       private
 
+      def normalize_arguments(arguments)
+        if arguments.last.is_a?(Hash)
+          options = arguments.pop
+          arguments.push(options.with_indifferent_access)
+        end
+
+        arguments
+      end
+
       def enqueued_in_block(block)
         before = @worker_class.jobs.dup
-        block.call
+        result = block.call
```

Comment:
```suggestion
        @result = block.call
```
Wondering if it's possible to expose this `result` in the `with`'s block somehow.